model/src/main/kotlin/licenses/LicenseConfiguration.kt

```diff
@@ -52,3 +52,5 @@ data class LicenseConfiguration(
         }
     }
 }
+
+fun LicenseConfiguration?.orEmpty(): LicenseConfiguration = if (this != null) this else LicenseConfiguration()
```

Comment:
Shorter: `this ?: LicenseConfiguration()`

Reply:
Done.